src/modules/tasks.js

```diff
@@ -0,0 +1,211 @@
+            <td class='col-2 ps-3'><button type='button' class="btn p-0 btn-light" data-bs-toggle="modal" data-bs-target="#Modal${index}">
+              Details</button><td>
+              <td class='col-1 ps-3'><button type='button' class='editTaskBtn btn p-0 btn-light'><i class="fa fa-wrench text-dark editTaskBtn"></i></button></td>
+            <td class='col-1 ps-3'><button type='button' class='deleteTaskBtn btn p-0 btn-light'><i class="fa fa-trash text-dark deleteTaskBtn"></i></button></td>
+
+          </tr>
+
+          <div class="modal fade" id="Modal${index}" tabindex="-1" aria-labelledby="${index}Label" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title fw-bold " id="${index}Label">${task.title}</h5>
+        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+      </div>
+      <div class="modal-body">
+        <p class="modalComment">Comment: ${task.comment} </p>
+        <p class="modalPrjct">Project: ${task.projectsN} </p>
+        <p>Priority: ${Flag(task.priority)}</p>
+        <p>Status: ${modalStatus(task.finished)} </p>
+        <p>Deadline: ${task.deadline} </p>
+      </div>
+    </div>
+  </div>
+</div>`;
+  });
+};
+
+const makeTaskSecOnClck = (value, selected) => {
+  if (!countProject) {
+    const tasksTitleDiv = document.createElement('div');
+    tasksTitleDiv.className = 'p-3';
+    tasksTitleDiv.innerHTML = `
+  <h2 class='prjct text-center mt-5'>Project: ${value}</h2>
+  <div class=" d-flex justify-content-center plus">
+    <h4 class="taskHeadline mt-1 me-5 mt-3 text-center">Tasks</h4>
+    <button type="button" class="pb-2 text-dark btn btn-light plus">
+      <i class="fa fa-plus plus"></i>
+    </button>
+  </div>`;
+
+    first.appendChild(tasksTitleDiv);
+    tasksToDom(selected);
+    countProject = true;
+  } else {
+    first.innerHTML = '';
+    tableArticle.innerHTML = '';
+    countProject = false;
+  }
+};
+
+const makeTaskSection = (value, selected) => {
+  first.innerHTML = '';
+  taskForm.innerHTML = '';
+  tableArticle.innerHTML = '';
+  const tasksTitleDiv = document.createElement('div');
+  tasksTitleDiv.className = 'p-3';
+  tasksTitleDiv.innerHTML = `
+  <h2 class='prjct text-center'>Project: ${value}</h2>
+  <div class=" d-flex justify-content-center plus">
+    <h4 class="taskHeadline mt-1 me-5 mt-3 text-center">Tasks</h4>
+    <button type="button" class="pb-2 text-dark btn btn-light plus">
+      <i class="fa fa-plus plus"></i>
+    </button>
+  </div>`;
+
+  first.appendChild(tasksTitleDiv);
+  tasksToDom(selected);
+  countProject = true;
+};
+
+const makeTaskForm = () => {
+  // const taskForm = document.createElement('form');
+  // taskForm.className = "taskForm border border-dark px-2 mx-5 ";
+  // taskForm.setAttribute('id', 'taskForm');
```

Comment:
Kindly remove the commented piece of code, use "git history" instead

Reply:
Done!